packages/medusa/src/loaders/helpers/routing/index.ts

```diff
@@ -332,21 +331,19 @@ export class RoutesLoader {
           }
 
           if (route.startsWith("/store")) {
-            config.shouldAppendCustomer = true
-
             if (shouldAddCors) {
               config.shouldAppendStoreCors = true
             }
+
+            if (shouldRequireAuth && route.startsWith("/store/customers/me")) {
```

Comment:
suggestion: /store/customers?

Reply:
@adrien2p I am sticking to what was already there. Also, the `/store/customers` endpoints apply a different type of authentication that allows for unregistered access

Reply:
would that mean that if you have GET store/customeres you would be able to list the customers?

Reply:
@adrien2p there is no `GET store/customers` handler defined, so you won't be able to do it. You can only create a new one.

Reply:
I am fine with keeping it, was just wondering ahah